lib/tbot/ssh_proxy.go

```diff
@@ -62,17 +63,27 @@ type ProxySSHConfig struct {
 // ProxySSH creates a local ssh proxy, dialing a node and transferring data through
 // stdin and stdout, to be used as an OpenSSH/PuTTY proxy command.
 func ProxySSH(ctx context.Context, proxyConfig ProxySSHConfig) error {
-	tshConfig, err := libclient.LoadTSHConfig(proxyConfig.TSHConfigPath)
-	if err != nil {
-		return trace.Wrap(err)
+	tshConfig := &libclient.TSHConfig{}
+	if proxyConfig.TSHConfigPath != "" {
+		var err error
+		tshConfig, err = libclient.LoadTSHConfig(proxyConfig.TSHConfigPath)
```

Comment:
When documenting proxy templates we should make it clear that only `proxy_templates` is currently supported, even if internally we're parsing the config file as a tsh config file.

Reply:
Agreed.